site/get-started.markdown

```diff
@@ -0,0 +1,143 @@
+
+Of other editors that have good Haskell extensions, the most popular ones are Vim and Emacs.
+
+## Running first lines of code
+
+We have everything set up, let's use it!
+
+`GHC` brings an interactive interpreter called `GHCi` together with it, which is great for playing with Haskell and trying things out, so let's give it a spin.
+So even though Haskell is a compiled language, it comes with an interpreter too -> how cool is that?
+
+Run `ghci`, which should start a new prompt for you.
+
+Let's do a simple calculation to check Haskell's computing capabilities:
+```
+> 6 + 3^2 * 4
+42
+```
+Hey, they call Haskell lazy, but that was quick!
+
+42 is a nice even number, but what about the even numbers after it? Let's get the first 10 even numbers after 42.
+```
+> take 10 $$ filter even [43..]
+[44,46,48,50,52,54,56,58,60,62]
+```
+
+What is the sum of those?
+```
+> sum it
+530
+```
+**NOTE**: We used a special feature of GHCi here, which is a special `it` variable that remembers the result of the last expression.
+
+Great, you got the first taste of Haskell! Now let's get to running a real program.
+
+## Writing your first Haskell program
+
+In your editor, create a new file named `hievb.hs`.
+
+Write the following in it:
+```
+main = do
+  putStrLn "Hi, everybody!"
+  putStrLn $$ "Pls look at my favorite odd numbers: " ++ filter odd [10..20]
+```
+
+You can now compile it with `ghc`, which will produce `hievb` binary that we will then run:
+```
+> ghc hievb.hs
+> ./hievb
+Hi, everybody!
+Pls look at my favorite odd numbers: [11,13,15,17,19]
+```
+
+There you go, you just wrote a short, polite program in Haskell!
+
+**TIP**: To interpret the source file directly, without producing any build artifacts, you can use the special `runghc` command like this:
+```
+> runghc hievb.hs
+Hi, everybody!
+Pls look at my favorite odd numbers: [11,13,15,17,19]
+```
+
+**GHCI TIP**: You can also load your file directly into `ghci`, which will enable you to play with any functions and other definitions you defined in it. So for our example, we can just load hievb.hs with ghci and then call the function `main` like this:
+```
+> ghci hievb.hs
+... Compiling ...
+> main
+Hi evb!
+```
+
+**PRO TIP**: Once you progress a bit more in your Haskell journey, you will very likely want to use `cabal` or `Stack` for building your projects, and not `ghc` directly.
+
+## Join the community
+
+By joining the Haskell community you will find a great place to ask for help and learn about new developments in the Haskell ecosystem.
+
+Some of the most popular communities are:
+
+ - [r/haskell](https://www.reddit.com/r/haskell/)
+ - [Haskell Discourse](https://discourse.haskell.org/)
+ - [https://wiki.haskell.org/IRC_channel](https://wiki.haskell.org/IRC_channel)
```

Comment:
A short description of each might be in order? I would even replace them with a link to the [Haskell Planetarium](https://haskell.pl-a.net/).

Reply:
Haskell Planetarium is cool, but it has so many links on it, too many for getting started I think. ALso, is it an official resource? What is wrong with us listing the 3 most popular places here where Haskellers are?

I didn't want to get into descriptions because I felt it is hard to write something without putting one option above the other and therefore getting subjective. Do you have any suggestions on what those descriptions could be?

Reply:
Having a few links seems like a good idea. Too many options would just be confusing for people.